test/e2e/encryption_at_rest_test.go

```diff
@@ -31,11 +31,21 @@ import (
 	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
 )
 
+const (
+	ClientID         = "AZURE_CLIENT_ID"
+	KeyVaultName     = "ako-kms-test"
+	ClientSecret     = "AZURE_CLIENT_SECRET" //#nosec G101 -- False positive; this is the env var, not the secret itself
```

Comment:
[nitpick]
These 3 are Azure specific, I wonder if we want to use a prefix in the name to make it more explicit where used?